types/2020-03-02/PaymentMethods.d.ts

```diff
@@ -126,7 +126,7 @@ declare module 'stripe' {
         exp_year: number;
 
         /**
-         * Uniquely identifies this particular card number. You can use this attribute to check whether two customers who've signed up with you are using the same card number, for example.
+         * Uniquely identifies this particular card number. You can use this attribute to check whether two customers who've signed up with you are using the same card number,for example. For payment methods that tokenize card information (Apple Pay, Google Pay), the tokenized number might be provided instead of the underlying card number.
```

Comment:
That comment change doesn't seem correct.

Reply:
Why isn't this correct? It's not greatly worded, I agree, but it just means we calculate the fingerprint based on the DPAN

Reply:
We should fix the comment to be clearer, but that's not a blocker for this PR.